src/axolotl/utils/config/models/input/v0_4_1/__init__.py

```diff
@@ -677,7 +704,11 @@ class Config:
     gpu_memory_limit: Optional[Union[int, str]] = None
     low_cpu_mem_usage: Optional[bool] = None
 
-    chat_template: Optional[ChatTemplate] = None
+    chat_template: Union[
+        ChatTemplate,
+        Annotated[str, StringConstraints(pattern="^tokenizer_default_fallback_")],
+    ] = ChatTemplate.chatml
```

Comment:
Does this `chatml` default also need to be changed to `tokenizer_default`?

I ran with the below model that has no chat_template in its config, and it defaulted to `chatml`.
```
base_model: mlx-community/gemma-2-2b

datasets:
  - path: LDJnr/Puffin
    type: chat_template
```